plugins/inputs/eventhub/README.md

```diff
@@ -0,0 +1,99 @@
+# Azure Event Hubs input plugin
+
+This plugin provides a consumer for use with Azure Event Hubs and Azure IoT Hub. The implementation is in essence a wrapper for [Microsoft Azure Event Hubs Client for Golang](https://github.com/Azure/azure-event-hubs-go).
+
+## Configuration
+
+```toml
+[[inputs.eventhub]]
```

Comment:
I'd like to rename this plugin `eventhub_consumer` to match the other queuing plugins.

Reply:
After merge